definitions/mlops-machine_learning_endpoint/definition.yml

```diff
@@ -37,7 +37,9 @@ synthesis:
 dashboardTemplates:
   newRelic:
     template: dashboard.json
+goldenTags:
+  - providerAccountId
+  - providerAccountName
+  - VariantName
 configuration:
-  entityExpirationTime: QUARTERLY
```

Comment:
synthesis entities should expire at some point, removing this makes them to be kept on the current default (eight days) was that expected?

Reply:
yes, that was the intention